Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3462,7 +3470,12 @@ public override void HighlightActElement(Act act)
                     //    highlightJavascript = "arguments[0].style.border='3px dashed red'";
                     //((IJavaScriptExecutor)Driver).ExecuteScript(highlightJavascript, new object[] { e });
                     //LastHighLightedElementInfo = elementInfo;
+                    //elementInfo.ElementObject = e;
+
+                    HTMLElementInfo elementInfo = new HTMLElementInfo();
```

Comment:
double check if we must learn the object here assuming in Highlight we will need to learn it again because we making it null

Reply:
learning the object isn't relevant. logic simplified/removed